db/migrate/20211130141653_remove_created_by_from_submissions.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveCreatedByFromSubmissions < ActiveRecord::Migration[6.1]
```

Comment:
Why do we need a remove migration?

Reply:
I first created the new column in camelCase, as `createdBy`, then removed it and added `created_by`. 

but I guess we can remove this migration now?